src/status_im/communities/core.cljs

```diff
@@ -307,6 +340,15 @@
                                    (log/error "failed to fetch communities" %)
                                    (re-frame/dispatch [::failed-to-fetch %]))}]})
 
+(rf/defn fetch-contract-communities
+  [_]
+  {:json-rpc/call [{:method     "wakuext_curatedCommunities"
+                    :params     []
+                    :on-success #(re-frame/dispatch [::fetched-contract-communities %])
```

Comment:
we're trying to avoid using namespaced keywords

Reply:
done